src/utils/isPlainObject.js

```diff
@@ -0,0 +1,3 @@
+export default function isPlainObject(obj) {
+  return obj ? typeof obj === 'object' && Object.getPrototypeOf(obj) === Object.prototype : false;
```

Comment:
A simple way to detect a pojo is to do Object.prototype.toString.call(object) === '[object Object]' or something like that